web/styles/subscriptions.css

```diff
@@ -787,25 +794,25 @@ h4.user_group_setting_subsection_title {
 
         .group-name-wrapper,
         .stream-name {
-            display: inline-block;
```

Comment:
Why this has been removed? I guess it should not have any effect. And if we want to remove it for being redundant then it can be done in separate commit.